Fix various typos not caught by codespell

luzpaz opened the pull request.

luzpaz commented:
Please review [966b2d4](https://github.com/godotengine/godot/pull/63281/commits/966b2d40ea08c592dc12cff252dbc024969dcd30) closely

akien-mga commented on core/input/gamecontrollerdb.txt:
```diff
@@ -439,7 +439,7 @@
 030000004c0500003713000000000000,PlayStation Vita,a:b1,b:b2,back:b8,dpdown:b13,dpleft:b15,dpright:b14,dpup:b12,leftshoulder:b4,leftx:a0,lefty:a1,rightshoulder:b5,rightx:a3,righty:a4,start:b9,x:b0,y:b3,platform:Windows,
 03000000d62000006dca000000000000,PowerA Pro Ex,a:b1,b:b2,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Windows,
-0300000062060000d570000000000000,PowerA PS3 Contoller,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Windows,
+0300000062060000d570000000000000,PowerA PS3 Controller,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Windows,
```
This should be fixed upstream: https://github.com/gabomdq/SDL_GameControllerDB

luzpaz replied:
Reverted and filed a PR upstream https://github.com/gabomdq/SDL_GameControllerDB/pull/600

akien-mga commented:
You say "typos **not caught** by codespell" but this looks like the typical class of typos that codespell does find. How did you find them?

luzpaz commented:
Running a HEAD version of codespell with multiple dictionaries (+ external dictionaries)

akien-mga reviewed (approved):
Looks good. Commits should be squashed.

luzpaz committed:
Fix various typos not caught by codespell

Revert upstream `core/input/gamecontrollerdb.txt`. Upstream fix: https://github.com/gabomdq/SDL_GameControllerDB/pull/600

luzpaz commented:
> Commits should be squashed.  

Done

akien-mga commented:
Thanks!
